app/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,45 @@
+<!--
+  ~ Copyright 2018 Google LLC
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     https://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<resources>
+    <string name="menu_filter_by_grow_zone">Nach Wachstumszone filtern</string>
+    <string name="plant_detail_image_content_description">Bild der Pflanze</string>
+    <string name="my_garden_title">Mein Garten</string>
+    <string name="plant_list_title">Pflanzenliste</string>
+    <string name="plant_details_title">Details zur Pflanze</string>
+    <string name="navigation_drawer_header_content_description">App Navigation</string>
+    <string name="added_plant_to_garden">Pflanze wurde zum Garten hinzugefügt</string>
+    <string name="garden_empty">Ihr Garten ist unbepflanzt</string>
+    <string name="planted_date">%1$s gepflanzt in %2$s</string>
+    <string name="menu_item_share_plant">Teilen</string>
+    <string name="share_text_plant">Schau dir die %s Pflanze in der App "Sunflower" an</string>
+
+    <string name="watering_needs_prefix">Wasserbedarf:</string>
+    <plurals name="watering_needs_suffix">
+        <item quantity="one">jeden Tag</item>
+        <item quantity="other">alle %d Tage</item>
+    </plurals>
+
+    <string name="watering_next_prefix">Zuletzt bewässert am %s</string>
```

Comment:
Let's keep this comment in so it's easier for developers to understand:

```<!-- String plurals and their related prefix / suffix strings -->```

Reply:
fine